### ISSUE TITLE
Logcollector sends duplicate data everytime the agent is restarted 

### ISSUE THREAD

**Issue by abohorqu** [wazuh/wazuh#7363]
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| 4.0.3-rev | LogCollector | Manager/Agent | Packages/Sources | Windows |

Team,

I've seen an issue where when configuring log data collection in a Wazuh agent for Windows, every time the agent is restarted it re-sends all log records from monitored log files, generating duplicate events in Kibana.

In this specific case, the issue is presented collecting Avast logs. The configuration looks like the following:
```
 <localfile>
    <location>C:\ProgramData\Avast Software\Avast\report\AntiRansomwareShield.txt</location>
    <log_format>syslog</log_format>
  </localfile>
  <localfile>
    <location>C:\ProgramData\AVG\Antivirus\report\WebShield.txt</location>
    <log_format>syslog</log_format>
  </localfile>
```

I have seen similar issues:

https://github.com/wazuh/wazuh/issues/6721
https://github.com/wazuh/wazuh/issues/4030

**Comment by juliancnn** [comment]
Hi @abohorqu,

This was fixed in this PR:  [Logcollector continue last line and multiline #6890](https://github.com/wazuh/wazuh/pull/6890) 

Regards,
Julian

**Comment by vikman90** [comment]
This issue should have been fixed by #9364.
